Doc/library/runpy.rst

```diff
@@ -106,11 +106,12 @@ The :mod:`runpy` module provides two functions:
    Execute the code at the named filesystem location and return the resulting
    module globals dictionary. As with a script name supplied to the CPython
    command line, the supplied path may refer to a Python source file, a
-   compiled bytecode file or a valid sys.path entry containing a ``__main__``
-   module (e.g. a zipfile containing a top-level ``__main__.py`` file).
+   compiled bytecode file or a valid ``sys.path`` entry containing a
+   ``__main__`` module (e.g. a zipfile containing a top-level ``__main__.py``
+   file).
```

Comment:
Style, but I prefer putting line breaks in-between distinct sections of the paragraph (commas, 'and', 'or', new clauses, parenthetical asides, etc) so that future edits can have a smaller diff.

```suggestion
   compiled bytecode file or a valid ``sys.path`` entry containing a
   ``__main__`` module
   (e.g. a zipfile containing a top-level ``__main__.py`` file).
```

Reply:
Done 👍